separate_instance/docker-compose.override.yml

```diff
@@ -0,0 +1,130 @@
+#      - ./admin:/app
+#
+  ########################################################
+  # BACKEND ##############################################
+  ########################################################
+  backend:
+    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+    image: gradido/backend:local-development
+    build:
+      target: development
+    networks:
+      - external-net
+      - internal-net
+    environment:
+      - NODE_ENV="development"
+    volumes:
+      # This makes sure the docker container has its own node modules.
+      # Therefore it is possible to have a different node version on the host machine
+      - backend_node_modules:/app/node_modules
+      - backend_database_node_modules:/database/node_modules
+      - backend_database_build:/database/build
+      # bind the local folder to the docker to allow live reload
+      - ./backend:/app
+      - ./database:/database
+
+  ########################################################
+  # DATABASE ##############################################
+  ########################################################
+  database:
+    # we always run on production here since else the service lingers
+    #  feel free to change this behaviour if it seems useful
+    #  Due to problems with the volume caching the built files
+    #  we changed this to test build. This keeps the service running.
+    # name the image so that it cannot be found in a DockerHub repository, otherwise it will not be built locally from the 'dockerfile' but pulled from there
+    image: gradido/database:local-test_up
+    build:
+      target: test_up
+    environment:
+      # Envs used in Dockerfile
+      # - DOCKER_WORKDIR="/app"
+      - BUILD_DATE
+      - BUILD_VERSION
+      - BUILD_COMMIT
+      - NODE_ENV="development"
+      - DB_HOST=mariadb
+      - DB_PORT=3307
+      - MYSQL_TCP_PORT=3307
+    volumes:
+      # This makes sure the docker container has its own node modules.
+      # Therefore it is possible to have a different node version on the host machine
+      - database_node_modules:/app/node_modules
+      - database_build:/app/build
+      # bind the local folder to the docker to allow live reload
+      - ./database:/app
+
+  #########################################################
+  ## MARIADB ##############################################
+  #########################################################
+  mariadb:
+    networks: 
+      - internal-net
+      - external-net
+
+#  #########################################################
+#  ## NGINX ################################################
+#  #########################################################
+#  # nginx:
+#  
+#  #########################################################
+#  ## PHPMYADMIN ###########################################
+#  #########################################################
+#  phpmyadmin:
```

Comment:
no need to comment this out you can start stuff specifically `docker-compose up mariadb database backend`

Reply:
yes you're right, but i had trouble with some modul-dependencies, which cause starting modules automatically, but i just not needed them for my scenario.
Commenting them out was the easiest solution...